esp/scm/ws_workunits.ecm

```diff
@@ -2156,6 +2156,7 @@ ESPStruct WUPropertiesToReturn
     bool AllHints(false);
     bool AllScopes(false);
     bool AllProperties(false);
+    [min_ver("1.76")] bool AllNotes(false);
```

Comment:
I assume this means either get them or not (no option to cherry pick some)?  Which I am pretty sure will be fine.

Reply:
Yes, at the moment, it is all or nothing as far as the notes.  We could add filtering later if required.

Reply:
Yes - at least for now.  If filtering would be useful, we can add later.